backend/lucene/src/main/java/org/hibernate/search/backend/lucene/search/projection/impl/FieldSearchProjectionImpl.java

```diff
@@ -14,7 +14,7 @@
 import org.hibernate.search.backend.lucene.types.converter.impl.LuceneFieldConverter;
 import org.hibernate.search.engine.search.query.spi.ProjectionHitCollector;
 
-class FieldSearchProjectionImpl<T> implements LuceneSearchProjection<T> {
+class FieldSearchProjectionImpl<T, U> implements LuceneSearchProjection<U> {
```

Comment:
Same here, I think `T` should be named `F` instead.

Reply:
Fixed.